build-for-web.sh

```diff
@@ -0,0 +1,18 @@
+
+# Add wasm32 target for compiler.
+rustup target add wasm32-unknown-unknown
+
+if ! command -v wasm-pack &> /dev/null
+then
+    echo "wasm-pack could not be found. Installing ..."
+    cargo install wasm-pack
+    exit
+fi
+
+# Set optimization flags
+export RUSTFLAGS="-C embed-bitcode=yes -C codegen-units=1 -C opt-level=3 --cfg web_sys_unstable_apis"
```

Comment:
I lean conservative when it comes to adding code; do we need `web_sys_unstable_apis`? I don't think we need `embed-bitcode` since  `yes` is [the default](https://doc.rust-lang.org/rustc/codegen-options/index.html). Finally, [this site](https://grafbase.com/blog/getting-started-with-rust-and-webassembly) thinks opt-level should be `s` or `z`?